pytripgui/controller/kernel_cont.py

```diff
@@ -1,360 +0,0 @@
-import logging
```

Comment:
@ljelen why this file is deleted ?

Reply:
I forgot to delete it in kernel_vc branch

Reply:
OK